### DISCUSSION TITLE
Old LoRA format giving noisy results

### DISCUSSION THREAD

**Discussion by nivedwho** [huggingface/diffusers#7911] [Q&A]
So I have an old LoRA trained using similar code as this repo - [lucataco/cog-sdxl-lcm](https://github.com/lucataco/cog-sdxl-lcm/tree/main). But when I load the lora.safetensors file using the latest diffusers version with peft, the results I get are noisy. Results are fine if I'm on older diffusers version. 

Can someone help me understand why this happens, and a potential solution to this?

**Comment by tolgacangoz** [comment]
Could you share a reproducible code snippet so that one can also try it directly?
